Makefile

```diff
@@ -0,0 +1,28 @@
+SHELL=bash
```

Comment:
Could we have a command to `npm publish` which i can use as owner of the NPM module?

Reply:
sure :)

Reply:
Added AWS access check.